### DISCUSSION TITLE
Is cypress visiting pages using HTTPS?

### DISCUSSION THREAD

**Discussion by VitalyAdweek** [cypress-io/cypress#16033] [Questions and Help] [answered]
We have some difficulties with scripts not being loaded and want to make sure that the following thing doesn't interfere with our problem
When we running tests there is a window where the runner executes it
It has HTTPS for the link in the middle and non HTTPS on the left side
Is Cypress using https visiting the website in our example or not?

<img width="1787" alt="Screen Shot 2021-04-16 at 10 03 04 AM" src="https://user-images.githubusercontent.com/82416786/115037243-471b7280-9e9c-11eb-979f-a9512d5c9003.png">

```
it.only( 'sample test', function() {
        cy.visit( 'https://google.com/' )
        cy.get( '.lnXdpd' ).should('be.visible')
    })
```

**Comment by flotwig** [comment, marked as answer]
Hi Vitaly, yes, all HTTPS requests sent by your test are really being sent over HTTPS. For more information on why it says "not secure", please see this FAQ: https://docs.cypress.io/faq/questions/using-cypress-faq#When-I-visit-my-site-directly-the-certificate-is-verified-however-the-browser-launched-through-Cypress-is-showing-it-as-Not-Secure-Why

**Reply by VitalyAdweek** [reply]
Thank you!

**Reply by bahmutov** [reply]
also see https://docs.cypress.io/guides/guides/web-security#Examples-of-what-Cypress-does-under-the-hood and Zach's answer is perfect
